test/functional/tests/fault_injection/test_fault_injection_interrupts.py

```diff
@@ -387,7 +375,7 @@ def test_interrupt_cache_mode_switch_parametrized(cache_mode, stop_percentage):
             if cache.get_cache_mode() != cache_mode:
                 TestRun.LOGGER.error("Cache mode should remain the same.")
 
-        with TestRun.step("Stop cache."):
+        with TestRun.step("Unmount core and stop cache."):
```

Comment:
there is no unmount 
sidenote: steps in this test are candidates for review and update